02/app.js

```diff
@@ -1,7 +1,47 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
+// const x = prompt('Podaj liczbę!');
+// const checkMultiplying = (num) => {
+//   for (let y = 1; y <= 9; y++) {
+//     console.log(`${x} x ${y} = ${x * y}`);
+//   }
+// };
```

Comment:
Raczej trzeba zwracać uwagę, że kod zakomentowany nie jest sprawdzany.
To ważne bo potem w kodzie "produkcyjnym" często pojawiają się komentarze, które przez lata tam zostają bo nikt nie wie czy są one potrzebne ;)